web/src/test/auto/headless/engine/main/headless/inputProcessor.spec.js

```diff
@@ -48,8 +48,8 @@ describe('InputProcessor', function() {
         assert.isUndefined(core.activeKeyboard); // No keyboard should be loaded yet.
         assert.isUndefined(core.activeModel);    // Same for the model.
 
-        // These checks are lifted from the keyboard-processor init checks found in
-        // common/web/keyboard-processor/tests/cases/basic-init.js.
+        // These checks are lifted from the keyboard init checks found in
+        // common/web/keyboard/tests/cases/basic-init.js.
```

Comment:
```suggestion
      // engine/js-processor/tests/cases/basic-init.js.
```

Assuming changes are made to the test location per my other comments.

Even if not, note that we're not in `common/web` anymore.